lychee-lib/src/helpers/request.rs

```diff
@@ -28,7 +27,7 @@ pub(crate) fn create(
         .into_iter()
         .map(|raw_uri| {
             let is_anchor = raw_uri.is_anchor();
-            let text = StrTendril::from(raw_uri.text.clone());
```

Comment:
changes in this file can prob be reverted but I didn't get why we use StrTendril here in the first place?

Reply:
That's just a remnant of using html5ever.